src/components/ui/card-enhanced.tsx

```
@@ -0,0 +1,172 @@
+    children,
+    ...props 
+  }, ref) => {
+    const getVariantStyles = () => {
+      switch (variant) {
+        case 'glass':
+          return {
+            ...getGlassStyle(glassVariant),
+            border: `1px solid ${getGlassStyle(glassVariant).border}`,
+          };
+        case 'gradient':
+          const gradientValue = gradient ? designTokens.gradients[gradient] : designTokens.gradients.primary;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)